jupyterhub/templates/hub/rbac.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.rbac.enabled -}}
+{{- if .Values.rbac.create -}}
```

Comment:
Ideally the name of the file describes the type of resource, so we will have a `hub/serviceaccount.yaml` which is fenced by `.Values. serviceAccount.create`, and a `hub/role.yaml` and a `hub/rolebinding.yaml` which are fenced by `.Values.rbac.create`